### ISSUE TITLE
Italian Localization

### ISSUE THREAD

**Issue by martinacocconcelli** [VincentAudibert/CanIDrive#116]
I am an Italian native speaker and I'm writing this because, along with another Italian translator, we would like to localize the app CanIDrive in Italian, since we think that it might be useful in order to contribute to tackling the social issue of drink-driving.

**Comment by VincentAudibert** [comment]
Sure, localizations are welcome 😀 and they happen in [/app/src/main/res](https://github.com/VincentAudibert/CanIDrive/tree/master/app/src/main/res), where `values` folder is the default (used for english) and `values-fr-rFR` is for french.

So what you can do is submit a new `values-it-rIT` folder (not sure about name of it, I will fix it if needed) with only the `strings.xml` file. You can compare english and french versions to see how the file is used.

**Comment by VincentAudibert** [comment]
Up @martinacocconcelli , forgot to mention you on initial reply...

**Comment by martinacocconcelli** [comment]
When localizing the presets, we noticed that some drinks (such as triple beer or soft cider) are not widely known in Italy. We thought that a possible solution to overcome this issue would be to find other well-known drinks with the same degrees as the original presets. Our solutions for the presets would be: vino rosso (red wine), birra bionda leggera (light beer), birra Ale forte (instead of triple beer),  Birra Radler (instead of soft cider), Martini and Whisky.

Another thing that we noticed  is the fact that when the user defines a new drink, they have to select the volume of the drink, which is measured in cl. Since in Italy it is more common to measure the volume of drinks in ml, would it be possible to change that feature? Is there anything that we can do about it?

**Comment by VincentAudibert** [comment]
Good idea to translate & adapt the preset !
I started a branch for the translation, if you have access (read sure, edit not so much) to the [destination file](https://github.com/VincentAudibert/CanIDrive/blob/locale-it/app/src/main/res/values-it-rIT/strings.xml), you can directly edit it online (then commit file).

For the cL, it is an old inconsistency from my part 😅 , corrected in version 0.2.8 pushed on PlayStore, should update on devices in about a week ...🐌

**Comment by martinacocconcelli** [comment]
Great! We confirm that the updated version shows the mL!
We have started a pull request in order to merge our translation with the main folder of the project, let us know if you can read it!

Thank you for your support and kind availability!

**Comment by VincentAudibert** [comment]
PR #117 merged !

@martinacocconcelli : are you ok having your name in version notes, and is there someone else involved that would do the same ? 😀

**Comment by martinacocconcelli** [comment]
Yeah that would be amazing! Could you please add Sara Giuliani as well? She is the other translator that worked on the translation of the app.

Thank you!! 😍

**Comment by VincentAudibert** [comment]
0.2.9 in the pipeline, thank you for contributing !

Thinking just now 😬, the [app presentation text on Playstore](https://play.google.com/store/apps/details?id=com.vaudibert.canidrive) is only in English & French, if you're ready to translate it too that would help again 🙏 .

**Comment by martinacocconcelli** [comment]
Thank you!! 

Considering the app presentation text we would love to translate it, should we paste the translation here or should we add a document as we did for the strings? 😀​

**Comment by martinacocconcelli** [comment]
Also, I updated the app localized in Italian and saw your acknowledgement (thank you for that 😍), but the presets are still in English 😅 
I've checked again in the strings document we translated and they are in Italian, is there something we can do about it?

**Comment by martinacocconcelli** [comment]
Thank you, I have just seen the update and everything seems to work properly, I also saw your acknowledgment in the pipeline!! 😍

As regards the app presentation text, we would love to translate it, should we paste the translation here or should we add a document as we did for the strings? 😀

**Comment by VincentAudibert** [comment]
You're welcome !

For the app presentation text, you can paste it here 👍 , I have nothing set up for versioning it now 😓.

**Comment by martinacocconcelli** [comment]
Hi Vincent! 
We have now completed the translation for the presentation of the app, I've pasted it here below for you!
Thank you again! 😃

Can I Drive ti permette di tracciare il consumo di alcol, al fine di stabilire se puoi guidare o no.

Nell’app ti verrà mostrato:
- lo stato di guida (OK, OK MA FAI ATTENZIONE, NO)
- una stima della concentrazione di alcol nel sangue
- l’orario in cui potrai guidare di nuovo
- l’orario in cui tornerai sobrio/a (0.0 g/L)

Il calcolo prende in considerazione:
- peso
- genere
- indicazione generica della tolleranza all’alcol
- stato legale del conducente (Paese, neopatentato, professionista)
- bevande (con quanto alcol e quando le hai consumate)

Se il Paese desiderato non è disponibile, seleziona ALTRO per impostare un nuovo valore limite.

**Comment by VincentAudibert** [comment]
🎉 🇮🇹  [app page online](https://play.google.com/store/apps/details?id=com.vaudibert.canidrive&hl=it&gl=US) 🇮🇹  🎉 

🙏

**Comment by martinacocconcelli** [comment]
Thank you!! 😍🎉
